bpf/tests/bpftest/bpf_test.go

```diff
@@ -290,299 +228,3 @@ func loadAndRunSpec(t *testing.T, entry fs.DirEntry, instrLog io.Writer) []*cove
 
 	return profiles
 }
-
-func loadAndPrepSpec(t *testing.T, elfPath string) *ebpf.CollectionSpec {
-	logger := hivetest.Logger(t)
-	spec, err := bpf.LoadCollectionSpec(logger, elfPath)
-	if err != nil {
-		t.Fatalf("load spec %s: %v", elfPath, err)
-	}
-
-	for _, m := range spec.Maps {
-		m.Pinning = ebpf.PinNone
-	}
-
-	for n, p := range spec.Programs {
-		switch p.Type {
-		case ebpf.XDP, ebpf.SchedACT, ebpf.SchedCLS:
-			continue
-		}
-
-		t.Logf("Skipping program '%s' of type '%s': BPF_PROG_RUN not supported", p.Name, p.Type)
```

Comment:
Should we still log this to provide some breadcrumbs for people running the tests as to why some programs are skipped? Just concerned that people might draw the wrong conclusions from the test runner silently skipping some progs.